charon/src/pretty/fmt_with_ctx.rs

```
@@ -823,6 +823,7 @@ impl<C: AstFormatter> FmtWithCtx<C> for RawConstantExpr {
                 format!("{}", f.fmt_with_ctx(ctx))
             }
             RawConstantExpr::RawMemory(bytes) => format!("RawMemory({bytes:?})"),
+            RawConstantExpr::Opaque(cause) => format!("Opaque({cause})"),
```

We should raise an error here too

---

oh oops, I indeed meant to add a comment wherever we create them :D